Date: 2017-12-19 19:51:49,804 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [26] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The INSERT statement conflicted with the FOREIGN KEY constraint "FK_tbl_Word_tbl_Language_LanguageId". The conflict occurred in database "DB_A2C13B_languageBuilderDb", table "dbo.tbl_Language", column 'Id'.
The statement has been terminated.
   at System.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__108_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.<ExecuteAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.<ExecuteAsync>d__32.MoveNext()
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.<ExecuteAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.<ExecuteAsync>d__10.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.<ExecuteAsync>d__7`2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.<SaveChangesAsync>d__61.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.<SaveChangesAsync>d__59.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.EntityFrameworkCore.DbContext.<SaveChangesAsync>d__48.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at LanguageBuilder.Data.Services.BaseRepository`2.<AddAsync>d__3.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\BaseRepository.cs:line 32
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Areas.Admin.Controllers.UploadController.<Words>d__9.MoveNext() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Areas\Admin\Controllers\UploadController.cs:line 120
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2017-12-22 10:29:03,566 

Host: Petar 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [13] 
Logger: LanguageBuilder.Web.Program 

Exception: System.Exception: test exception
   at LanguageBuilder.Web.Controllers.HomeController.Index() in D:\PStoyanov\repo\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\HomeController.cs:line 22
 
Message: test exception 

Query: (null) 



--------------------------------------



Date: 2017-12-24 10:46:57,627 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [10] 
Logger: LanguageBuilder.Web.Program 

Exception: System.NotSupportedException: Only public methods can be invoked in the background.
   at Hangfire.Common.Job.Validate(Type type, String typeParameterName, MethodInfo method, String methodParameterName, Int32 argumentCount, String argumentParameterName)
   at Hangfire.Common.Job..ctor(Type type, MethodInfo method, Object[] args)
   at Hangfire.Common.Job.FromExpression(LambdaExpression methodCall, Type explicitType)
   at Hangfire.BackgroundJobClientExtensions.Create(IBackgroundJobClient client, Expression`1 methodCall, IState state)
   at LanguageBuilder.Web.Areas.Admin.Controllers.UploadController.<Words>d__11.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Areas\Admin\Controllers\UploadController.cs:line 122
 
Message: Only public methods can be invoked in the background. 

Query: (null) 



--------------------------------------



Date: 2017-12-26 21:17:31,233 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [8] 
Logger: LanguageBuilder.Web.Program 

Exception: System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at System.UriBuilder.get_Uri()
   at LanguageBuilder.Web.Infrastructure.Extensions.HttpRequestExtensions.UrlReferrer(HttpRequest request) in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Infrastructure\Extensions\HttpRequestExtensions.cs:line 18
   at LanguageBuilder.Web.Controllers.TranslationsController.<DeleteConfirmed>d__14.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\TranslationsController.cs:line 226
 
Message: Invalid URI: The hostname could not be parsed. 

Query: (null) 



--------------------------------------



Date: 2017-12-26 21:18:59,855 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [54] 
Logger: LanguageBuilder.Web.Program 

Exception: System.UriFormatException: Invalid URI: The hostname could not be parsed.
   at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at System.UriBuilder.get_Uri()
   at LanguageBuilder.Web.Infrastructure.Extensions.HttpRequestExtensions.UrlReferrer(HttpRequest request) in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Infrastructure\Extensions\HttpRequestExtensions.cs:line 18
   at LanguageBuilder.Web.Controllers.TranslationsController.<DeleteConfirmed>d__14.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\TranslationsController.cs:line 226
 
Message: Invalid URI: The hostname could not be parsed. 

Query: (null) 



--------------------------------------



Date: 2017-12-26 21:25:51,245 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [45] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The INSERT statement conflicted with the FOREIGN KEY constraint "FK_tbl_Translation_tbl_Word_SourceWordId". The conflict occurred in database "DB_A2C13B_languageBuilderDb", table "dbo.tbl_Word", column 'Id'.
The INSERT statement conflicted with the FOREIGN KEY constraint "FK_tbl_UserWord_tbl_Word_WordId". The conflict occurred in database "DB_A2C13B_languageBuilderDb", table "dbo.tbl_Word", column 'Id'.
The statement has been terminated.
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(Tuple`2 parameters)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ExecutionStrategyExtensions.Execute[TState,TResult](IExecutionStrategy strategy, TState state, Func`2 operation)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at LanguageBuilder.Data.LanguageBuilderDbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Data\LanguageBuilderDbContext.cs:line 88
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at LanguageBuilder.Services.Implementations.WordsService.<AddWordsWithTranslationWithUserConnectionAsync>d__11.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\WordsService.cs:line 138
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.TranslationsController.<Create>d__8.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\TranslationsController.cs:line 126
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2017-12-26 21:30:05,892 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [8] 
Logger: LanguageBuilder.Web.Program 

Exception: Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: The INSERT statement conflicted with the FOREIGN KEY constraint "FK_tbl_Translation_tbl_Word_SourceWordId". The conflict occurred in database "DB_A2C13B_languageBuilderDb", table "dbo.tbl_Word", column 'Id'.
The INSERT statement conflicted with the FOREIGN KEY constraint "FK_tbl_UserWord_tbl_Word_WordId". The conflict occurred in database "DB_A2C13B_languageBuilderDb", table "dbo.tbl_Word", column 'Id'.
The statement has been terminated.
The statement has been terminated.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, SqlDataReader ds)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean asyncWrite, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior)
   at System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   at System.Data.Common.DbCommand.ExecuteReader()
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(Tuple`2 parameters)
   at Microsoft.EntityFrameworkCore.Storage.Internal.SqlServerExecutionStrategy.Execute[TState,TResult](TState state, Func`3 operation, Func`3 verifySucceeded)
   at Microsoft.EntityFrameworkCore.ExecutionStrategyExtensions.Execute[TState,TResult](IExecutionStrategy strategy, TState state, Func`2 operation)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at LanguageBuilder.Data.LanguageBuilderDbContext.SaveChanges(Boolean acceptAllChangesOnSuccess) in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Data\LanguageBuilderDbContext.cs:line 88
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at LanguageBuilder.Services.Implementations.WordsService.<AddWordsWithTranslationWithUserConnectionAsync>d__11.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\WordsService.cs:line 138
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.TranslationsController.<Create>d__8.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\TranslationsController.cs:line 126
 
Message: An error occurred while updating the entries. See the inner exception for details. 

Query: (null) 



--------------------------------------



Date: 2018-01-08 13:21:12,521 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [5] 
Logger: LanguageBuilder.Web.Program 

Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at LanguageBuilder.Services.Implementations.WordsService.<AddWordsWithTranslationWithUserConnectionAsync>d__11.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\WordsService.cs:line 122
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.TranslationsController.<Create>d__8.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\TranslationsController.cs:line 123
 
Message: Object reference not set to an instance of an object. 

Query: (null) 



--------------------------------------



Date: 2018-01-08 13:22:41,583 

Host: DESKTOP-CQAS2KA 
Url: NOT AVAILABLEspnet-request{URL} 
Query string: NOT AVAILABLEspnet-request{QUERY_STRING} 
Request method: NOT AVAILABLEspnet-request{REQUEST_METHOD} 
Referer: NOT AVAILABLEspnet-request{HTTP_REFERER} 
User: NOT AVAILABLE 
User IP: NOT AVAILABLEspnet-request{REMOTE_ADDR} 
User agent: NOT AVAILABLEspnet-request{HTTP_USER_AGENT} 

Thread: [10] 
Logger: LanguageBuilder.Web.Program 

Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at LanguageBuilder.Services.Implementations.WordsService.<AddWordsWithTranslationWithUserConnectionAsync>d__11.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Services\Implementations\WordsService.cs:line 122
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at LanguageBuilder.Web.Controllers.TranslationsController.<Create>d__8.MoveNext() in C:\Users\pesho\Google Drive\Programming\GitHub\ASP.NET-Core-MVC\LanguageBuilder\LanguageBuilder.Web\Controllers\TranslationsController.cs:line 123
 
Message: Object reference not set to an instance of an object. 

Query: (null) 



--------------------------------------



